src/reducers/global.ts

```diff
@@ -491,10 +496,11 @@ export const global = (state = defaultState, action: any) => {
         ),
         cladeSliderField: cladeSliderField,
         cladeSliderValue: formatMrcaSliderOptionValue(tree, cladeSliderField),
-        mrca: tree,
+        mrca: tree, // TODO should be informed by mrcaParam from search params
         metadataCensus: { ...state.metadataCensus, ...treeMetadata },
         // Added portion for Fetch aspect starts here
         uploadModalOpen: true,
+        pathogen: pathogenParam || defaultState.pathogen,
```

Comment:
```suggestion
        pathogen: pathogenParam && Object.keys(pathogenParameters).includes(pathogenParam.toLowerCase()) ? pathogenParam.toLowerCase() : "other",
```

Reply:
check if the pathogen they supplied is supported (i.e., we have default values prepared for this pathogen). Otherwise default to `other` (i.e., we know it's a pathogen not in the list so pop open the calculator for them.

Reply:
Okay, I pushed up a commit to resolve, although I moved the lowercasing to its own var.